lexicons/tools/ozone/moderation/queryStatuses.json

```diff
@@ -8,7 +8,16 @@
       "parameters": {
         "type": "params",
         "properties": {
-          "subject": { "type": "string", "format": "uri" },
+          "forAccount": {
```

Comment:
kind of feels like this should be a boolean? named `allAccountSubjects` or something. you could give either a DID as `subject`, or an AT-URI. either way all subjects for the account would be returned if this was true.

Reply:
makes sense. went with the term `includeAllUserRecords` since we're using that for queryEvents.